azure/pas-full-network.json

```diff
@@ -594,7 +645,45 @@
             }
           },
           {
-            "name": "DenyAllOut",
+            "name": "Allow1858VaultOut",
+            "properties": {
+              "protocol": "TCP",
+              "sourcePortRange": "*",
```

Comment:
Why source port range is *? Do we want to allow using another source port for communication with the vault?